### ISSUE TITLE
ClassNotFoundException: jdk.internal.misc.Unsafe

### ISSUE THREAD

**Issue by byzy** [netty/netty#6901]
### OS version Linux debian8 3.16.0-4-amd64 1 SMP Debian 3.16.43-2 (2017-04-30) x86_64 GNU/Linux

```
16:33:07.450 [main] DEBUG io.netty.util.internal.PlatformDependent0 - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable
java.lang.ClassNotFoundException: jdk.internal.misc.Unsafe
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at io.netty.util.internal.PlatformDependent0$6.run(PlatformDependent0.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at io.netty.util.internal.PlatformDependent0.<clinit>(PlatformDependent0.java:288)
	at io.netty.util.internal.PlatformDependent.getSystemClassLoader(PlatformDependent.java:895)
	at io.netty.util.internal.PlatformDependent.isAndroid0(PlatformDependent.java:919)
	at io.netty.util.internal.PlatformDependent.<clinit>(PlatformDependent.java:70)
	at io.netty.util.ConstantPool.<init>(ConstantPool.java:32)
	at io.netty.util.Signal$1.<init>(Signal.java:27)
	at io.netty.util.Signal.<clinit>(Signal.java:27)
	at io.netty.util.concurrent.DefaultPromise.<clinit>(DefaultPromise.java:43)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:36)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:58)
	at io.netty.channel.MultithreadEventLoopGroup.<init>(MultithreadEventLoopGroup.java:52)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:87)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:82)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:63)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:51)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:43)
```
how will not find jdk.internal.misc.Unsafe class

**Comment by normanmaurer** [comment]
Fixed in 4.1.12

**Comment by Scottmitch** [comment]
@byzy - Please update to 4.1.12. If the issue has not been fixed please re-open with new stack trace info.
